src/unstract/adapters/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "0.5.0"
+__version__ = "0.5.1"
```

Comment:
@vishnuszipstack this version bump is not needed - `unstract-adapters==0.5.0` has not been [published to PyPI](https://pypi.org/project/unstract-adapters/) yet. We could essentially treat this as part of 0.5.0 version itself

Reply:
done @chandrasekharan-zipstack